tests/Integration/Database/DatabaseEloquentModelAttributeCastingTest.php

```diff
@@ -142,6 +142,8 @@ public function testOneWayCasting()
     {
         $model = new TestEloquentModelWithAttributeCast;
 
+        $this->assertEquals(null, $model->password);
```

Comment:
Can use `$this->assertNull($model->password);`

Reply:
Thanks!